### ISSUE TITLE
Download URL on PyPI broken

### ISSUE THREAD

**Issue by cdeil** [astropy/astropy#5739]
I noticed that at https://pypi.python.org/pypi/astropy/1.3 it says:
> Download URL: https://pypi.python.org/packages/source/a/astropy/astropy-1.3.tar.gz

but that doesn't seem to exist.

Clicking it in the browser for me shows:
```
This XML file does not appear to have any style information associated with it. The document tree is shown below.
<Error>
<Code>NoSuchKey</Code>
<Message>The specified key does not exist.</Message>
<Key>source/a/astropy/astropy-1.3.tar.gz</Key>
<RequestId>9AC53FACF6B12E81</RequestId>
<HostId>
kV1lJ8XDMJmRqWXuTs1Kc1F07Vet/AcLyQI5yrTu5sW3n4LCto5yuwHxzsRoNwn4az2EFISZgjQ=
</HostId>
</Error>
```
and trying to fetch it gives this error:
```
$ wget https://pypi.python.org/packages/source/a/astropy/astropy-1.3.tar.gz
--2017-01-26 08:15:53--  https://pypi.python.org/packages/source/a/astropy/astropy-1.3.tar.gz
Resolving pypi.python.org (pypi.python.org)... 151.101.112.223, 2a04:4e42:1b::223
Connecting to pypi.python.org (pypi.python.org)|151.101.112.223|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2017-01-26 08:15:53 ERROR 404: Not Found.
```

I didn't check other Astropy releases or packages and try to track down if the "bug" comes from Astropy or is a regression on PyPI.

@eteq @astrofrog or anyone knowledgeable about PyPI releases - can you please have a look?

**Comment by astrofrog** [comment]
Maybe we should simply remove the download URL from the metadata? I wonder what purpose it really serves since the downloads are listed above, and there isn't a single download anyway?

**Comment by saimn** [comment]
See also #5474 and my comment there (https://github.com/astropy/astropy/pull/5474#issuecomment-261482324): I should have open an issue but I forgot about it, sorry ! So 👍 to simply remove `download_url`.

**Comment by cdeil** [comment]
@saimn - Based on your explanation at https://github.com/astropy/astropy/pull/5474#issuecomment-261482324 I agree that removing `download_url` is the way to go.

Anyone willing to check Astropy-affiliated packages also and make pull requests?
I can attach a commit for Astropy here if you want. Let me know.

**Comment by astrofrog** [comment]
I've been meaning for a while to make a generic framework to make modifications to repositories in bulk and open pull requests. This issue would be a nice application, so I'm willing to open the pull requests for this if there are no objections to removing download_url. It might take me a couple of days to do this.

**Comment by cdeil** [comment]
@astrofrog - I don't know if any / how many affiliated packages have `download_url` set. It's not here or in Gammapy: https://github.com/astropy/package-template/blob/master/setup.py#L124

So a tool for batch modification might not be needed here.
But a tool for batch search of Astropy-affiliated packages would.
Maybe it exists or can be done just with the Github global search feature with the web or the API?

**Comment by astrofrog** [comment]
@cdeil - I agree it might not be needed here in the end, but it seems like a good excuse to try making something - there are other changes across affiliated packages I'd like to make :)

**Comment by cdeil** [comment]
@astrofrog - OK, so you'll take care of it. Thanks!

I'm unsubscribing from this issue now.
:-)

**Comment by bsipocz** [comment]
@astrofrog - I also have half baked scripts to do bulk edits on affiliates (ugly bash hacks, so they're not really shareable, but I meant to come up with something better), so if you could use any help ping me.

**Comment by bsipocz** [comment]
I've checked the affiliates and basically none of those using the template have a ``download_url`` listed in their ``setup.py``. (No fancy script to do the search, just the good old ``find . -name setup.cfg|xargs grep download_url|grep -v astropy_helpers``)

**Comment by pllim** [comment]
Addressed in #5740.
